### ISSUE TITLE
docker cli client is stuck/hanging after docker run

### ISSUE THREAD

**Issue by A-Shevchenko** [rancher-sandbox/rancher-desktop#1448]
### Rancher Desktop Version

1.0.0

### Rancher Desktop K8s Version

1.22.6

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

Windows

### Operating System / Build Version

Windows 10 Pro 1909

### What CPU architecture are you using?

x64

### Actual Behavior

When using simple docker run commands from windows shell (powershell or cmd) like `docker run ubuntu whoami` - docker client is stuck after actual output so I can't get back my shell:
```
PS D:\> docker run ubuntu whoami
root
```
Ctrl+C doesn't help to get shell back. Container is not running anymore (checked with `docker ps`)

Same behaviour with -i flag

However if I run it with -it - it's a bit different:
```
PS D:\> docker run -it ubuntu whoami
root
failed to resize tty, using default size
```
it's stuck as well, but I can get shell back by Ctrl+C

Though if I run it with -t only - it's the same initial problem, stuck with no way to get it back (and "failed to resize tty, using default size" is displayed)

Probably it looks like not related to Rancher Desktop, but I didn't have that problem with Docker Desktop which was uninstalled just before.

### Steps to Reproduce

`docker run ubuntu whoami`

### Result

Docker client is stuck, can't get shell back

### Expected Behavior

Shell is back after docker run

**Comment by evertonlperes** [comment]
Hi @A-Shevchenko 
We already fixed this issue and it will be available on the next release: `1.0.1`.
Please, feel free to try the dev package: https://github.com/rancher-sandbox/rancher-desktop/actions/runs/1780533861

Also tagging the similar issues for your reference:
https://github.com/rancher-sandbox/rancher-desktop/issues/1171
https://github.com/rancher-sandbox/rancher-desktop/issues/1361

Closing this issue. Feel free to reopen if you still facing the problem.

**Comment by jandubois** [comment]
> We already fixed this issue and it will be available on the next release: `1.0.1`.

This will not be part of the `1.0.1` patch release, but rather of the next release, `1.1.0`.

**Comment by jandubois** [comment]
> This will not be part of the `1.0.1` patch release, but rather of the next release, `1.1.0`.

We changed our mind and still got it into `1.0.1`, so it has been released now. Sorry for the confusion!
